src/h5web/providers/mock/metadata.ts

```diff
@@ -117,44 +113,3 @@ export const mockMetadata = makeNxGroup(mockDomain, 'NXroot', {
     ]),
   ],
 });
-
-/* ------------------ */
-/* ----- VALUES ----- */
```

Comment:
`makeDataset` needs access to `mockValues`, so I had to move the mock values to a separate file to avoid cyclic imports.